Makefile

```diff
@@ -0,0 +1,20 @@
+# developer Makefile for repeated tasks
+# 
+
+.PHONY: clean
+
+nb:
+	docker run -it --rm  -p 8888:8888 -p 4000:4000 -v ${PWD}:/home/jovyan/work darribas/gds_dev:4.0
+
+term:
+	docker run -it --rm  -p 8888:8888 -p 4000:4000 -v ${PWD}:/home/jovyan/work darribas/gds_dev:4.0 sh -c "/bin/bash"
```

Comment:
the official Jupyter stacks have a convenience script that drops you into a terminal by calling `start.sh`:

```
docker run -it --rm  -p 8888:8888 -p 4000:4000 -v ${PWD}:/home/jovyan/work darribas/gds_dev:4.0 start.sh
```